letsencrypt-auto-source/letsencrypt-auto

```diff
@@ -812,10 +795,22 @@ elif [ -f /etc/redhat-release ]; then
       BootstrapMessage "Legacy Red Hat-based OSes that will use Python3"
       BootstrapRpmPython3Legacy
     }
+    EnablePython36SCL() {
```

Comment:
nit: What do you think about defining this function in `rpm_python3_legacy.sh` before `BootstrapRpmPython3Legacy`? It somewhat matches the pattern in `rpm_common_base.sh` and helps logically split up the code a bit in my opinion.

Reply:
I agree. Done.